demo/package.json

```diff
@@ -24,7 +24,7 @@
 		"fs-extra": "^9.0.0",
 		"html-webpack-plugin": "^4.5.2",
 		"interpolate-html-plugin": "^4.0.0",
-		"node-sass": "^4.14.1",
+		"sass": "^1.44.0",
```

Comment:
@petrKavulok You want to add `react-content-loader` to the packages.json of demo app